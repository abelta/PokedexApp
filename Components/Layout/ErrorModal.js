import React from 'react';
import PropTypes from 'prop-types';
import { Modal, Text } from 'react-native';

const ErrorModal = ({ children }) => (
  <Modal>
    <Text>{children}</Text>
  </Modal>
);

ErrorModal.propTypes = {
  children: PropTypes.node.isRequired,
};

export default ErrorModal;
