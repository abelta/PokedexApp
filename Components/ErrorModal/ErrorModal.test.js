import React from 'react';
import renderer from 'react-test-renderer';
import ErrorModal from '.';

describe('ErrorModal', () => {
  const tree = renderer.create(<ErrorModal>some text</ErrorModal>).toJSON();
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
