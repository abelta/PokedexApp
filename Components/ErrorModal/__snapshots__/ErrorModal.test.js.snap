// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorModal renders correctly 1`] = `
<Modal
  hardwareAccelerated={false}
  visible={true}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    some text
  </Text>
</Modal>
`;
